statusHandler/presenterStatusHandler.go

```diff
@@ -0,0 +1,296 @@
+package statusHandler
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+//maxLogLines is used to specify how many lines of logs need to store in slice
+var maxLogLines = 100
+
+// PresenterStatusHandler will be used when an AppStatusHandler is required, but another one isn't necessary or available
+type PresenterStatusHandler struct {
```

Comment:
Might spawn this struct over multiple files (blockInfoGetters.go, chainInfoGetters.go...)

Reply:
Done.